### DISCUSSION TITLE
[Adonis V5] Incomplete GET parameter

### DISCUSSION THREAD

**Discussion by melokki** [adonisjs/core#1709] [Help] [answered]
Hello, 
I'm having a small problem and I can't find a solution to it.
From my frontend, I am using fullcallendar the get some events from my api (adonis). Fullcalendar sends to my backend the following request `appointments?start=2020-09-28T00:00:00+03:00&end=2020-11-09T00:00:00+02:00`
I my backend, when I am getting the `start` parameter I am missing the `+` sign in my date. I get `2020-09-28T00:00:00 03:00` instead of `2020-09-28T00:00:00+03:00`

Is this normal? How can I get the correct date in my backend?

**Comment by thetutlage** [comment]
You need to url encode your query string params.

**Comment by melokki** [comment, marked as answer]
yes, that was my first thought, but the thing is I don't have access to those parameters before the request is sent.

I managed to solve this after all using the [events as a function](https://fullcalendar.io/docs/v3/events-function) options from fullcalendar, now I get in my backend a full string, and I can use luxon from there to play with my dates.

Thank you very much for your replay :upside_down_face:
